caa-admin/recipes/default.rb

```diff
@@ -20,22 +27,17 @@
   force_reinstalls true
 end
 
-include_recipe "daemontools"
-service "svscan" do
-  action :start
-  provider Chef::Provider::Service::Upstart
+template '/home/caaadmin/svc-caa-admin/devel.cfg' do
+  source 'devel.cfg.erb'
+  owner 'caaadmin'
+  group 'caaadmin'
+  mode '0755'
+  notifies :hup, 'daemontools_service[caa-admin]'
```

Comment:
I'm not sure if caa-admin responds to SIGHUP at all. It's safe to restart it though because it's only an internal admin tool.